django/researchdata/models.py

```diff
@@ -232,7 +232,7 @@ class Letter(models.Model):
     transcription_plain = models.TextField(blank=True, null=True)
     transcription_normalized = models.TextField(blank=True, null=True)
     sent_date_year = models.IntegerField(blank=True, null=True,
-                                         validators=[MinValueValidator(1500), MaxValueValidator(2000)])
```

Comment:
What does the field represent exactly?

Reply:
historical dates are difficult to work with, as the letters they receive may just have the month or the year or the day or different combinations of these, so I can't use a normal datefield as a datefield requires a full date (which they rare have). so each part (year, month, day) are split into different fields. so the year field is just an integer. I originally set the min value as 1500 but they've got older dates they want to include and asked the min value to be reduced.